### PR TITLE
Bugfix/mds 3420

### PR DESCRIPTION
**Pull request by mhuseinov**
First part
- fix for status when completing a technical review for an application
The second part with data clean up will be in a separate pr

### CODE REVIEW

File: migrations/sql/V2021.03.04.12.22__update_now_application_progress_status.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/*
```
Comment (exzizt) [inline migrations/sql/V2021.03.04.12.22__update_now_application_progress_status.sql:1]: Could you mark with a comment the difference between each `referal_complete_applications`? Also, not a big deal but "referal" should be spelled "referral". 🤓  Can we remove the commented-out code as well?

Comment (mhuseinov) [reply migrations/sql/V2021.03.04.12.22__update_now_application_progress_status.sql:1]: The pseudo-code(the condition in each 'if' statement explains the difference in the 'with' clause) above each block of update statement explains the logic used to determine which applications should be set to referral.

**Review by exzizt — DISMISSED** [review]
Nice work :)

**Review by mmumby — APPROVED** [review]
🔥

**Review by nayr974 — APPROVED** [review]

**Pull request merged**